airline-data/src/main/scala/com/patson/init/AirportFeaturePatcher.scala

```diff
@@ -73,6 +73,7 @@ object AirportFeaturePatcher {
         "CUN" -> 32, //Cancun
         "ORY" -> 32, //Paris Orly
         "GUM" -> 31, //Guam
+        "BDA" -> 35, //Bermuda
```

Comment:
35 seems way too strong, i dun think it should be more than Cancun and Guam...

Not really sure how big of a vacation spot is Bermuda, I searched Hotels.com and Bermuda gives like 35 result, while other countries in Carribean, for example Aruba - gives 288 , while Aruba i only gave it a 11 charm and Barbados - 382 hotels, which is only with 11 charm as well :)

Reply:
I didn't think too much about it, I saw Tenerife, the Canary islands, Pape'ete (no idea where's that), Punta Cana (no idea either) and figured Bermuda would be around the same ballpark, I didn't know you had a metric like that. 
So, ~8?